CODE_OF_CONDUCT.md

```diff
@@ -1,6 +1,80 @@
+    - [Example](#example)
+  - [How to make a Code Of Conduct Report](#how-to-make-a-code-of-conduct-report)
+    - [Information to include in your report](#information-to-include-in-your-report)
+  - [Code of Conduct Reporting Guide](#code-of-conduct-reporting-guide)
+  - [What happens after you file a report?](#what-happens-after-you-file-a-report)
+    - [Example](#example-1)
+  - [What if your report concerns a possible violation by a moderator?](#what-if-your-report-concerns-a-possible-violation-by-a-moderator)
+  - [How CoC Reports are handled by moderators](#how-coc-reports-are-handled-by-moderators)
+  - [Accountability for board members and moderators](#accountability-for-board-members-and-moderators)
+  - [Definition of Terms](#definition-of-terms)
+  - [Original Source Links](#original-source-links)
+    - [Reference Codes of Conduct](#reference-codes-of-conduct)
+
+# Guiding principles of the CoC
+
+## Goals for the Code of Conduct
+
+The Code of Conduct should
+
+* Be Inclusive.
+* Use welcoming and inclusive language.
+* Protect vulnerable members and establish plans to deal with incidents.
+* Emphasize the importance of kindness and empathy.
+* Apply to community members, board members, and contributors.
+* Provide a description of how members may be held accountable and how conflicts will be managed transparently.
+* Allow people accusing someone of sexual assault or behaviour of a similarly serious nature to be heard fully, without prejudice, and as soon as possible.
+* Provide constructive ways to deal with conflict in the community (meetings, polls...).
+
+We strive to reduce barriers to access of our organisation, our software and our community, as well as to dismantle any ableist assumptions and practices in our process. We recognize that there is no one solution for universal accessibility, and strive to provide alternatives and choices whenever possible.
+
+The stakeholders of this project pledge to institute and abide by a consistent, transparent and accountable human-centered resolution process with a low bar to entry that allows every stakeholder a voice on decisions to do with structural changes and features of both the project's organization and software protocol.
+
+If you support an idea, you must contribute towards its implementation.
+If you reject an idea, you must contribute to an alternative solution to the problem it tries to solve.
+
+## CoC Outline Brainstorm
+
+* Principles of positive participation and communication.
+* Where does this CoC apply?
+* Who does this CoC apply to?
+* Prohibited behavior.
+* When something happens - Give the chance to improve.
+* Examples of how promote and teach respectful, effective communication.
+* How to make a Code Of Conduct Report.
+* Outline how CoC reports will be handled in a transparent and fair way.
+* How CoC Reports are handled by moderators.
+* Accountability for board members of moderators.
+* Definitions of terms.
+
```

Comment:
Remove everything above here. ( I forgot to tell you before you started doing this )